test/config-startup.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of iotagent-json
+ *
+ * iotagent-json is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * iotagent-json is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with iotagent-json.
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::[contacto@tid.es]
+ */
+var config = {};
+
+config.mqtt = {
+    host: 'localhost',
+    port: 1883,
+    defaultKey: '1234',
+    thinkingThingsPlugin: true
+};
+
+config.iota = {
+    logLevel: 'FATAL',
```

Comment:
This is a test file, not the default configuration. This information is only used to check the environment variables really change the information.